Makefile.in

```diff
@@ -186,8 +186,8 @@ am__DIST_COMMON = $(srcdir)/Makefile.in \
 	$(top_srcdir)/config/install-sh $(top_srcdir)/config/ltmain.sh \
 	$(top_srcdir)/config/make.inc.in $(top_srcdir)/config/missing \
 	config/compile config/config.guess config/config.rpath \
-	config/config.sub config/install-sh config/ltmain.sh \
-	config/missing
+	config/config.sub config/depcomp config/install-sh \
+	config/ltmain.sh config/missing
```

Comment:
This list of shell scripts seems to change based on the version of autoconf used. Since we have a `config/depcomp` script I believe we should include it here.

Reply:
At some point we made some effort to ensure consistent autotools automatically.

Reply:
Good question - these changes show up consistently for me when I use 2.69, which is the version we suggest using in `ibtk/bootstrap`.